onnxruntime/core/graph/contrib_ops/contrib_defs.cc

```diff
@@ -785,6 +785,46 @@ ONNX_MS_OPERATOR_SET_SCHEMA(BiasDropout, 1,
                                   }
                                 }));
 
+ONNX_MS_OPERATOR_SET_SCHEMA(
+    BitmaskBiasDropout, 1,
```

Comment:
The file just has inconsistent intent between the 1st part and 2nd part. The intent around is not pretty and every easy to exceed the 120 column limit. Instead of following the old intent space-by-space during the coding, I think the eventual way is to format the whole file. But this file is too big, so I just selected my part and "format selected" to avoid too many changes.